kitsune/users/api.py

```diff
@@ -288,10 +239,8 @@ class Meta(ProfileSerializer.Meta):
 
 
 class ProfileViewSet(
-    mixins.CreateModelMixin,
     mixins.RetrieveModelMixin,
     mixins.ListModelMixin,
-    mixins.UpdateModelMixin,
```

Comment:
Should we remove the profile update API as well?

Reply:
I am not sure I understand your question.